intl.emu

```diff
@@ -11,6 +11,37 @@
     </p>
   </emu-note>
 
+  <emu-clause id="sup-properties-of-the-number-prototype-object">
+    <h1>Properties of the Number Prototype Object</h1>
+
+    <p>
+      The following definition(s) refer to the abstract operation thisNumberValue as defined in es2024, <emu-xref href="#sec-properties-of-the-number-prototype-object"></emu-xref>.
+    </p>
+
+    <emu-clause id="sup-decimal128.prototype.tolocalestring">
+      <h1>Number.prototype.toLocaleString ( [ _locales_ [ , _options_ ] ] )</h1>
+
+      <p>
+        This definition supersedes the definition provided in es2024, <emu-xref href="#sec-decimal128.prototype.tolocalestring"></emu-xref>.
+      </p>
+
+      <p>
+        When the `toLocaleString` method is called with optional arguments _locales_ and _options_, the following steps are taken:
+      </p>
+
+      <emu-alg>
+        1. Let _numberFormat_ be ? Construct(%Intl.NumberFormat%, &laquo; _locales_, _options_ &raquo;).
```

Comment:
Generally the this-value should be processed first, then the arguments, so move this after any processing of the this-value that could throw (such as RequireInternalSlot if you accept my other suggestion)

Reply:
Ah, I see. Good idea. Done!